File: 1120.py
k1, k2 = map(int, input().split())
l1, l2 = map(int, input().split())
 
if l1 - k1 == l2 - k2:
    print("YES")
elif (l1-k1) * -1 == l2-k2:
    print("YES")
elif l1 - k1 == (l2-k2 * -1):
    print("YES")
elif (l1-k1) * -1 == (l2-k2) * -1:
    print("YES")
else:
    print("NO")

File: 1120.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
 
int main(){
 
    int x1, y1, x2, y2;
    scanf("%d %d %d %d", &x1, &y1, &x2, &y2);
 
    if(abs(x2-x1) == abs(y2-y1)){
        printf("YES");
    }else{
        printf("NO");
    }
 
    return 0;
}
